tests/general/InvalidLinkBearTest.py

```diff
@@ -255,10 +257,26 @@ def response(status_code, *args, **kwargs):
             self.assertEqual([x.message
                               for x in list(uut.run('file', file_contents,
                                                     network_timeout=nt))], [])
+
+            with self.assertLogs(logging.getLogger()) as cm:
+                self.assertEqual([x.message
+                                  for x in list(uut.run('file', file_contents,
+                                                        timeout=20))], [])
+            self.assertEqual(cm.output,
```

Comment:
hmm indent this please to line up so it's in the the with block. It's generally a bad idea to use a cm var outside of the with. (let me know if it's jsut github being stupid)